pinot-broker/src/main/java/org/apache/pinot/broker/querylog/QueryLogger.java

```diff
@@ -88,9 +87,12 @@ public void log(QueryLogParams params) {
       queryLogBuilder.append(',');
     }
 
-    // always log the query last - don't add this to the QueryLogEntry enum
+    // update: stop logging the query itself here, as it will already be logged in the request handler
+    /*
     queryLogBuilder.append("query=")
         .append(StringUtils.substring(params._requestContext.getQuery(), 0, _maxQueryLengthToLog));
+    */
+    queryLogBuilder.setLength(queryLogBuilder.length() - 1);
```

Comment:
I think there may be users using this log already. I think it is better to log the query twice than breaking compatibility

Reply:
I share the same concern. Also, if logs get eventually compressed, having the same query twice should have a smaller impact.

Reverted for now, the query will be logged twice.